### PR TITLE
Multi-project build: core, storm, samza.

### PR DESCRIPTION
**Pull request by gianm**
- Publish tranquility-core, tranquility-samza, tranquility-storm as separate artifacts
- Bump version to 0.6.0

### CODE REVIEW

**Comment by fjy** [comment]
:+1:

File: build.sbt [hunk 1]
```diff
@@ -57,43 +59,28 @@ val finagleVersion = "6.25.0"
...
-  "com.metamx" % "java-util" % "0.26.14" exclude("log4j", "log4j") force()
-)
-
-libraryDependencies ++= Seq(
-  "io.netty" % "netty" % "3.10.1.Final" force()
-)
-
-libraryDependencies ++= Seq(
+  "com.metamx" % "java-util" % "0.27.4" exclude("log4j", "log4j") force(),
+  "io.netty" % "netty" % "3.10.1.Final" force(),
```
Comment (xvrl) [inline build.sbt]: might as well update to 3.10.4 ?

File: build.sbt [hunk 2]
```diff
@@ -30,18 +28,22 @@ pomExtra := (
...
       <developer>
         <name>Gian Merlino</name>
-        <organization>Metamarkets Group Inc.</organization>
-        <organizationUrl>https://www.metamarkets.com</organizationUrl>
+        <organization>Druid Project</organization>
+        <organizationUrl>http://druid.io/</organizationUrl>
       </developer>
     </developers>)
 
+parallelExecution in ThisBuild := false
```
Comment (xvrl) [inline build.sbt]: can we add some comments to explain why we need those options?

File: core/src/main/scala/com/metamx/tranquility/beam/ClusteredBeam.scala [hunk 3]
```diff
@@ -1,18 +1,20 @@
 /*
- * Tranquility.
```
Comment (xvrl) [inline core/src/main/scala/com/metamx/tranquility/beam/ClusteredBeam.scala]: can we put the header changes into a separate commit?

Comment (xvrl) [inline core/src/main/scala/com/metamx/tranquility/beam/ClusteredBeam.scala]: mainly because it makes it possible to track file renames if we do it that way.

Comment (gianm) [inline core/src/main/scala/com/metamx/tranquility/beam/ClusteredBeam.scala]: ok

File: build.sbt (hunk 2 above)
Comment (gianm) [inline build.sbt]: sure

File: build.sbt (hunk 1 above)
Comment (gianm) [inline build.sbt]: sure

**Comment by gianm** [comment]
@xvrl pushed those changes

**Comment by gianm** [comment]
new PR #33
